codalab/model/bundle_model.py

```diff
@@ -817,7 +817,9 @@ def resume_bundle(self, bundle, bundle_update, row, user_id, worker_id, connecti
         metadata_update = {
             'run_status': bundle_update['run_status'],
             'last_updated': int(time.time()),
-            'time': time.time() - bundle_update['start_time'],
```

Comment:
Basically this was the line causing all the issues, trying to recompute time used with each update.
Now the time is only calculated by the worker. I guess technically someone could write an adversarial worker that always reports 0 time usage, but if we want to do server-side time calculation, that would be a bigger change and probably on a separate effort.